pwsh/buildkimageformats.ps1

```diff
@@ -1,26 +1,34 @@
 #!/usr/bin/env pwsh
 
-$qtVersion = [version]((qmake --version -split '\n')[1] -split ' ')[3]
+$qtVersion = [version](qmake -query QT_VERSION)
 Write-Host "Detected Qt Version $qtVersion"
 
-$kde_vers = 'v5.116.0'
+$kfGitRef =
+    $qtVersion -ge [version]'6.6.0' ? 'v6.10.0' :
+    $qtVersion -ge [version]'6.5.0' ? 'v6.8.0' :
+    'v5.116.0'
```

Comment:
CMakeLists.txt in KImageFormats specifies REQUIRED_QT_VERSION (i.e. minimum) as 6.5 in KDE Frameworks 6.0 through 6.8. Minimum Qt version was raised to 6.6 in KDE Frameworks 6.9 - present. So this is picking the newest KDE Frameworks version compatible with the desired Qt version.